docs/excel/build-your-first-excel-add-in.md

```diff
@@ -251,52 +256,62 @@ You’ll start by creating a web app using HTML and JQuery. Then you'll create a
         <Capability Name="Workbook" />
       </Capabilities>
       <DefaultSettings>
-        <SourceLocation DefaultValue="\\MyShare\QuarterlySalesReport\Home.html" />
+        <SourceLocation DefaultValue="~remoteAppUrl/QuarterlySalesReportWeb/Home.html" />
       </DefaultSettings>
       <Permissions>ReadWriteDocument</Permissions>
     </OfficeApp>
 ```
 
-7-  Generate a GUID using an online generator of your choice. Then, replace the value in the **Id** tag shown in the previous step with that GUID.
+2-  Generate a GUID using an online generator of your choice. Then, replace the value of the **Id** element shown in the previous step with that GUID.
+
+3-	Save the XML manifest file. 
 
-8-	Save all the files. You’ve now written your first Excel add-in.
+Now that you've created the web app and the XML manifest file, you’ve officially written your first Excel add-in.
 
 ### Try it out
 
-The simplest way to deploy and test your add-in is to copy the files to a network share.
+#### Deploy the web app and update the manifest
```

Comment:
Similar comment here - do we need "Try it out", or can the subheadings be elevated to the H2 level?